controller/helper/process-survey-instances.js

```diff
@@ -22,15 +20,36 @@ function processSurveyInstances (surveys) {
 /**
  * Takes in a Survey Instances and get the % time left to be shown on complience chart
  * @param {Object} surveys - list of survey instances
+ * @returns {Object} processed list of datetimes
+ */
+function pickDates (surveys) {
+    const dates = surveys.map((survey) => {
+        return moment(new Date(survey.startTime).toISOString()).utc().format('MM/DD/YYYY HH:mm');
+    });
+
+    const lastDate = moment(new Date(surveys[0]
+        .dateCompleted)
+        .toISOString()).utc().format('MM/DD/YYYY HH:mm');
+    const missingValue = -1;
+
+    if (dates.indexOf(lastDate) <= missingValue) {
```

Comment:
> No. I take the completedDate from the first record. This date will be same for all the records, so I chose to pick it from the first one.

okay

> This is to make sure the chart will contain all the dates for 5-10 weeks.

How does it do this?
It adds one date at the end if it can't find the date in some list.